src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.cs

```diff
@@ -21,16 +21,25 @@ static Startup()
 
         internal static string ManagedProfilerDirectory { get; }
 
-        private static bool TryLoadManagedAssembly()
+        private static void TryLoadManagedAssembly()
         {
             try
             {
-                Assembly.Load(new AssemblyName("Datadog.Trace.ClrProfiler.Managed, Version=1.11.1.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb"));
-                return true;
+                var assembly = Assembly.Load("Datadog.Trace.ClrProfiler.Managed, Version=1.11.1.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb");
+
+                if (assembly != null)
+                {
+                    // call method Datadog.Trace.ClrProfiler.Instrumentation.Initialize()
+                    var type = assembly.GetType("Datadog.Trace.ClrProfiler.Instrumentation", throwOnError: false);
+                    var method = type?.GetRuntimeMethod("Initialize", parameters: new Type[0]);
+                    method?.Invoke(obj: null, parameters: null);
+                }
             }
-            catch
+            catch (Exception e)
             {
-                return false;
+                Console.Error.WriteLine();
```

Comment:
Will it always be safe to write to `Console.Error`?

Reply:
Good point. Should we just ignore the exception?

Reply:
I think so. It would be nice if we could log this with our Tracer logs or our Profiler logs but this could be pretty tricky.

Reply:
Changes this in ef8e56b6e9cda4165626432a79f6d2706e3f094b to ignore exceptions.